lora-phy/src/lorawan_radio.rs

```diff
@@ -29,22 +34,36 @@ where
         Self {
             lora,
             rx_pkt_params: None,
+            rx_window_offset_ms: DEFAULT_RX_WINDOW_OFFSET_MS,
+            rx_window_duration_ms: DEFAULT_RX_WINDOW_DURATION_MS,
         }
     }
 }
 
+impl<RK, DLY, const P: u8, const G: i8> LorawanRadio<RK, DLY, P, G>
+where
+    RK: RadioKind,
+    DLY: DelayNs,
+{
+    pub fn set_rx_window_offset_ms(&mut self, offset: i32) {
+        self.rx_window_offset_ms = offset;
+    }
+    pub fn set_rx_window_duration_ms(&mut self, duration: u32) {
+        self.rx_window_duration_ms = duration;
+    }
+}
+
 /// Provide the timing values for boards supported by the external lora-phy crate
 impl<RK, DLY, const P: u8, const G: i8> Timings for LorawanRadio<RK, DLY, P, G>
 where
     RK: RadioKind,
     DLY: DelayNs,
 {
-    // TODO: Use a struct with Default or Option fallback?
     fn get_rx_window_offset_ms(&self) -> i32 {
```

Comment:
Could you add a comment on what this offset and duration actually does?

Reply:
The comment exists on the trait definition already: https://github.com/lora-rs/lora-rs/blob/main/lorawan-device/src/lib.rs#L61-L70

Reply:
Whoops, didn't see that